dask/array/tests/test_reductions.py

```diff
@@ -954,6 +967,7 @@ def test_nanquantile_all_nan():
         assert_eq(
             da.nanquantile(darr, 0.75, axis=-1), np.nanquantile(arr, 0.75, axis=-1)
         )
+        assert_eq(da.percentile(darr, 0.75, axis=-1), np.percentile(arr, 0.75, axis=-1))
```

Comment:
Would splitting this off into its own test (`test_quantile_all_nan`) be more explicit?

Reply:
Not sure, they are calling the same thing under the hood so that makes kind of sense to me, merging this for now but happy to make a follow up